Wykonane zadanie

a-dreszler opened the pull request.

a-dreszler committed:
Robocza wersja programu

a-dreszler committed:
Ostateczna wersja aplikacji

marcinkunert reviewed (changes requested):
Code review: https://javastartbootcamp.slack.com/archives/C04LP8YSVQX/p1680971388427049

- Drukowanie do konsoli przenieść do `Phone`

a-dreszler committed:
Poprawki

a-dreszler committed:
Ostateczna wersja programu.
Zmiany:
 > Przeniesiono odpowiedzialność za wyświetlanie rezultatu wykonania operacji do klasy Phone
 > Metoda call nazywa się teraz getCallDuration i zwraca długość rozmowy. Na podstawie tej długości rozmowy wyświetlany jest teraz komunikat
 > Metody sendSms i sendMms nazywają się teraz canSendSms i canSendMms i zwracają true lub false w zależności czy możliwe jest wysłanie smsa/mmsa
 > Metoda printAccountState w klasie Phone od teraz pobiera stan konta z kontraktów za pomocą metody getAccountState
